src/main/resources/db/migration/V3__Create_klage_schema.sql

```diff
@@ -1,34 +1,61 @@
+CREATE TYPE id_type AS ENUM ('PERSON', 'ORGANISASJON', 'VIRKSOMHET');
```

Comment:
Hva er dette for postgres-magi?

Reply:
Det er strengt tatt bare ekstra validering. Vi kan godt droppe det og kun ha den valideringen i koden. Kanskje like lurt.